avutil/avutil_test.go

```diff
@@ -845,3 +845,13 @@ func TestPixelFormatDescriptor_ComponentCount(t *testing.T) {
 		t.Fatalf("[TestPixelFormatDescriptor_ComponentCount] count=%d, NG expected=%d", count, 3)
 	}
 }
+
+func TestMotionVectorFromFrameSideData(t *testing.T) {
```

Comment:
Can you make a separate function `TestFrameSideData` where you call all its `func`s?